api/handlers.go

```diff
@@ -782,6 +792,41 @@ func proxyVolume(c *context, w http.ResponseWriter, r *http.Request) {
 	httpError(w, fmt.Sprintf("No such volume: %s", name), http.StatusNotFound)
 }
 
+// Proxy network to container operations, including connect/disconnect request
+func proxyNetworkContainerOperation(c *context, w http.ResponseWriter, r *http.Request) {
+	var networkid = mux.Vars(r)["networkid"]
+	network := c.cluster.Networks().Uniq().Get(networkid)
+	if network == nil {
+		httpError(w, fmt.Sprintf("No such network: %s", networkid), http.StatusNotFound)
+		return
+	}
+	// Set the network ID in the proxied URL path.
+	r.URL.Path = strings.Replace(r.URL.Path, networkid, network.ID, 1)
+
+	// make a copy of r.Body
+	buf, _ := ioutil.ReadAll(r.Body)
+	copy1 := bufReader{bytes.NewBuffer(buf)}
```

Comment:
We don't have a copy2 or any other variable with copy. This var name can drop the number. Maybe `bodyCopy`?